uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -1169,9 +1169,15 @@ sub registerScanIntoDB {
         ########################################################
         ### update mri_acquisition_dates table #################
         ########################################################
+        my $acquisition_date = undef;
```

Comment:
Use of the `//` operator maybe? As in

```perl
my $acquisition_date = $tarchiveInfo->{'DateAcquired'} 
    // $${minc_file}->getParameter('AcquisitionDate') 
    // undef;
```

I am open to arguments though...

Reply:
done. Thanks for the suggestion. I'll try to remember that in the future :).